device-types/Eaton/Tripp-Lite-SMART500RT1U.yaml

```diff
@@ -0,0 +1,24 @@
+---
+manufacturer: Eaton
+model: Tripp Lite SMART500RT1U
+slug: eaton-tripp-lite-smart500rt1u
+u_height: 1
+is_full_depth: false
+front_image: true
+rear_image: true
```

Comment:
1. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. This appears to be listed in the datasheet (15.80 lbs or 7.17 kg)